src/renderer/components/settings-electron.tsx

```diff
@@ -199,9 +201,22 @@ export const ElectronSettings = observer(
         </ButtonGroup>
       );
     }
+    private filterSection(): JSX.Element {
+      const { appState } = this.props;
+      return (
+        <FormGroup inline={true} label="Filters:">
```

Comment:
To keep the design similar to `Channels` formgroup, `inline={true}` can be safely removed.